web/src/SyncerListPage.js

```diff
@@ -205,12 +216,13 @@ class SyncerListPage extends BaseListPage {
         title: i18next.t("general:Action"),
         dataIndex: '',
         key: 'op',
-        width: '170px',
+        width: '240px',
         fixed: (Setting.isMobile()) ? "false" : "right",
         render: (text, record, index) => {
           return (
             <div>
-              <Button style={{marginTop: '10px', marginBottom: '10px', marginRight: '10px'}} type="primary" onClick={() => this.props.history.push(`/syncers/${record.name}`)}>{i18next.t("general:Edit")}</Button>
+              <Button style={{marginTop: '10px', marginBottom: '10px', marginRight: '10px'}} type="primary" onClick={() => this.syncUsers(index)}>{i18next.t("general:Sync")}</Button>
```

Comment:
Plz also add this button to the edit page

Reply:
we have to write `runSyncer` function twice to do that, maybe it's now cleaner